public/app/plugins/panel/heatmap/tooltip/tooltipUtils.ts

```diff
@@ -0,0 +1,94 @@
+import { Field, FieldType, formattedValueToString } from '@grafana/data';
```

Comment:
nit: but we probably could rename this file to just `utils` given it is in the `tooltip/` directory